.NET/buildtools/checkSpec.ps1

```diff
@@ -0,0 +1,129 @@
+# Check for empty and duplicate inputs in Specs
+Param ([string]$rootFolder = (get-item $(Get-Location).Path).parent.FullName)
+
+$SpecsFile = Join-Path -Path $rootFolder -ChildPath "Specs"
+$filePattern = "*.json"
+
+$global:totalEmpty = 0
+$global:totalDuplicate = 0
+$global:emptyFileList = New-Object System.Collections.Generic.List[string]
+$global:duplicateFileList = New-Object System.Collections.Hashtable
+
+function SpecInfo()
+{
+    foreach ($file in $input) 
+    {   
+        $parentName = $file.FullName | Split-Path -parent | Split-Path -leaf
+        $typeFolder = $file.FullName | Split-Path -parent | Split-Path -parent | Split-Path -leaf
+		$ret = Get-Content $file.FullName -encoding utf8 | ConvertFrom-Json
```

Comment:
Please convert all tabs to spaces. :)

Reply:
$ret can be renamed to something like $contents to better represent what it contains.